### DISCUSSION TITLE
现在的ProLayout 不会在路由页面第一次加载，根据资源加载情况自动Loading了吗

### DISCUSSION THREAD

**Discussion by Lands-1203** [umijs/umi#10857] [Q&A]
请问怎么开启这个功能？
![image](https://user-images.githubusercontent.com/61341868/227896248-ca60bf3c-ee4f-42f0-acce-ebe1af1a2029.png)

**Comment by Lands-1203** [comment]
这是官方线上地址访问的效果
![image](https://user-images.githubusercontent.com/61341868/227898825-ed61a640-07cd-49c8-8081-8beb64aa2ac2.png)
这是拉取项目后本地跑起来的效果：layout的自动根据资源加载情况的loading不存在了
![image](https://user-images.githubusercontent.com/61341868/227898917-3846beab-788d-439b-9a2d-abae23b665aa.png)

**Reply by fz6m** [reply]
全局 loading 组件在 `src/loading.tsx` 编写即可，欢迎 PR 给 max 模板添加 `loading.tsx` 。

**Reply by Lands-1203** [reply]
好的谢谢

**Comment by Lands-1203** [comment]
现在在umijs中，内容区的高度为0，也就会出现闪屏的问题

**Reply by fz6m** [reply]
你说得对，container 的部分是没有高度的，需要里面撑起来，你可以调试看看是 antpro 的 ProLayout 样式需要调整还是 max 模板的样式需要改，我理解自己把内容撑起来也是可以的。
